invenio.cfg

```diff
@@ -550,3 +550,11 @@ VOCABULARIES_NAMES_SCHEMES = {
     "lcds": {"label": _("CDS"), "validator": schemes.is_legacy_cds, "datacite": "CDS"},
 }
 """Names allowed identifier schemes."""
+
+
+# Invenio Stats
+# =============
+
+# We override the templates to add new fields needed for the migrated statistic events
+_APP_RDM_STATS_EVENTS["file-download"]["templates"] = "cds_rdm.stats.templates.events.file_download"
```

Comment:
I guess we will need to remember on the deployment too? should we note it down somewhere?

Reply:
you mean update of the mapping? I will do it yes